CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ## 🔧 Tech
 
+* Remove useless demo timeline
```

Comment:
I feel 'useless' is a bit of a harsh term, maybe 'obsolete' or 'unused' would be better. Obviously a nitpick